engine-server/src/modules/extension_account/extension_account.utils.ts

```diff
@@ -1,3 +1,3 @@
 /** Copyright (c) 2023, Poozle, all rights reserved. **/
 
-export function getSpecForExtensionAccount(extensionAccountName: string) {}
```

Comment:
why are we commenting this line?

Reply:
This is failing when building the server and we are not using it anywhere, afaik. Am I missing something here